packages/shell-api/src/index.ts

```diff
@@ -42,7 +42,9 @@ export {
   Cursor,
   CursorIterationResult,
   Database,
+  DatabaseImpl,
```

Comment:
Do we want to be exporting the `Impl`s here? I'd expect those to be used only in this package and then we use the `Collection`/`Database` outside.